psi4/src/psi4/lib3index/df_helper.cc

```diff
@@ -2062,54 +2313,64 @@ void DF_Helper::JK_disk(std::vector<SharedMatrix> Cleft, std::vector<SharedMatri
     std::shared_ptr<BasisSet> zero = BasisSet::zero_ao_basis_set();
     std::shared_ptr<IntegralFactory> rifactory(new IntegralFactory(aux_, zero, primary_, primary_));
     std::vector<std::shared_ptr<TwoBodyAOInt>> eri(nthreads_);
-    #pragma omp parallel private(rank) num_threads(nthreads_)
-    {
-        #ifdef _OPENMP
+    
+    // manual cache alignment..(poor std vectors..) (assumes cache size is 64bytes)
+    size_t remainder = nao*nao;
```

Comment:
`remainder` is a C++ [function](http://www.cplusplus.com/reference/cmath/remainder/).